app/javascript/dashboard/assets/scss/_woot.scss

```diff
@@ -34,6 +34,8 @@
 @import 'plugins/multiselect';
 @import 'plugins/dropdown';
 
+@import 'flag-icons/css/flag-icons.min.css';
```

Comment:
Let us import this in the component rather than globally.

Reply:
@pranavrajs Done